include/effectmanager.h

```diff
@@ -153,9 +153,19 @@ class  EffectManager : public IJSONSerializable
         ClearEffects();
     }
 
-    std::shared_ptr<GFXBase> GetBaseGraphics()
+    // SetTempEffect - Sets a temporary effect to be played until remote changes it.
+    //                 The effect must have already had its Init() function called.
+
+    void SetTempEffect(std::shared_ptr<LEDStripEffect> effect)
+    {
+        _tempEffect = effect;
+    }
+
+    // GetBaseGraphics - Returns the vector of GFXBase objects that the effects use to draw
+    
+    std::vector<std::shared_ptr<GFXBase>> & GetBaseGraphics()
     {
-        return _gfx[0];
+        return _gfx;
     }

```

Comment:
I now return the entire array of graphics rather than just the first so that the VU meter effect can draw onto both spokes of the spiral lamp, which has two channels